src/back/glsl/mod.rs

```diff
@@ -777,6 +818,10 @@ impl<'a, W: Write> Writer<'a, W> {
         handle: Handle<crate::GlobalVariable>,
         global: &crate::GlobalVariable,
     ) -> BackendResult {
+        if self.info.get_entry_point(self.entry_point_idx as usize)[handle].is_empty() {
```

Comment:
it would be easier to do this check at the caller side, since they already have `ep_info` by hand

Reply:
Updated locally.